### ISSUE TITLE
mpld3 attempt relegated to new branch

### ISSUE THREAD

**Issue by danfeldman90** [Robox-AI/Humpback_Whale_ID#4]
Given the difficulty I've had with trying to use mpld3 with the GitHub uploads, I've relegated it to a separate branch and merged in the remaining changes to master so we can skip it and keep going. But it'll be in the new branch if we want to take it back up later. 

In it's current state, it runs on my machine perfectly, but doesn't show up in GitHub (and the markdown from README.md doesn't work at all).

**Comment by allieoop** [comment]
When you say the markdown from README.md doesn't work at all, do you mean when you export to markdown and save it as README.md it doesn't look correct?

**Comment by allieoop** [comment]
I just ran the code in the mpld3_attempt branch and it works on my machine too. I just had to install mpld3 and pillow, which I added to our requirements.txt.

**Comment by danfeldman90** [comment]
Yeah, you can look at the README file to see what I mean. It just shows the html code instead of doing things. I first tried using the export only, then I saved the figure as html and tried that code in the README and in both cases it didn't work.

Good catch, I forgot to update the requirements file!
